cmd/sync.go

```diff
@@ -318,25 +322,24 @@ func (cca *cookedSyncCmdArgs) reportScanningProgress(lcm common.LifecycleMgr, th
 	})
 }
 
-func (cca *cookedSyncCmdArgs) getJsonOfSyncJobSummary(summary common.ListSyncJobSummaryResponse) string {
-	// TODO figure out if deletions should be done by the enumeration engine or not
-	// TODO if not, remove this so that we get the proper number from the ste
-	summary.DeleteTotalTransfers = cca.getDeletionCount()
-	summary.DeleteTransfersCompleted = cca.getDeletionCount()
-	jsonOutput, err := json.Marshal(summary)
+func (cca *cookedSyncCmdArgs) getJsonOfSyncJobSummary(summary common.ListJobSummaryResponse) string {
+	wrapped := common.ListSyncJobSummaryResponse{ListJobSummaryResponse: summary}
+	wrapped.DeleteTotalTransfers = cca.getDeletionCount()
```

Comment:
Will this be a breaking change for any callers, in terms of how things are named in the JSON?  Not saying that would stop us doing it, just that if so we should be sure to list it in changelog.md (even tho there are not likely to be many users of this yet, I'm guessing)